benefit/data/sheets/CaseTypeTab/CaseTypeTab.json

```diff
@@ -233,6 +233,7 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedChildElement", "TabFieldDisplayOrder": 7},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedChildDisabled", "TabFieldDisplayOrder": 8},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedLimitedWork", "TabFieldDisplayOrder": 9},
+
```

Comment:
Empty new line, does this break formatting / sheets at all?

Reply:
Its done to break out the sections. It wont break it, lots of the file has them.